packages/office-addin-dev-settings/src/dev-settings-windows.ts

```diff
@@ -70,8 +71,8 @@ export async function enableRuntimeLogging(path: string): Promise<void> {
 }
 
 export function getDeveloperSettingsRegistryKey(addinId: string): registry.RegistryKey {
-  if (!addinId) { throw new Error("addinId is required."); }
-  if (typeof addinId !== "string") { throw new Error("addinId should be a string."); }
+  if (!addinId) { throw new ExpectedError("addinId is required."); }
```

Comment:
I wonder if we should make this more specific to let the user know that it's "manifest" ID is missing?

Reply:
This is not neccessarily from the manifest. It is in an input parameter to the function.  We should keep both checks and I think the existing messaging is OK. If you want to tweak it you can make them:

"The addIn parameter is required." and "The addIn parameter should be a string."

Reply:
I guess this new message is a little bit more clear and I'm already setup to change this message, so I will update it to be more complete as you suggested